recipes/common/Makefile.common

```diff
@@ -101,7 +102,7 @@ bootc: quadlet
 
 .PHONY: bootc-run
 bootc-run:
-	podman run -d --rm --name $(APP)-bootc -p 8080:8501 --privileged \
+	podman run -d --rm --name $(APP)-bootc -p 8080:$(PORT) --privileged \
```

Comment:
You need to define a default for PORT somewhere?

Reply:
Ports are specific to recipe and get defined at the top of those makefiles. This way there will always be a PORT defined.